NDTensors/src/lib/Unwrap/test/runtests.jl

```diff
@@ -1,11 +1,27 @@
-using Test
+using Test: @testset, @test, @test_broken
 using NDTensors.Unwrap
-using NDTensors
-using LinearAlgebra
+using NDTensors: NDTensors, mul!!
+using LinearAlgebra:
+  LinearAlgebra,
+  Adjoint,
+  Diagonal,
+  Hermitian,
+  Symmetric,
+  Transpose,
+  eigen,
+  mul!,
+  norm,
+  qr,
+  svd
 using GPUArraysCore: @allowscalar
-
-include(joinpath(pkgdir(NDTensors), "test", "device_list.jl"))
-@testset "Testing Unwrap $dev, $elt" for dev in devices_list(ARGS),
+include(
+  joinpath(
+    pathof(NDTensors)[1:(end - 16)], "test", "NDTensorsTestUtils", "NDTensorsTestUtils.jl"
```

Comment:
The code `pathof(NDTensors)[1:(end - 16)]` looks pretty sketchy, could you explain that?

Reply:
I was using the wrong function; I meant to be using `pkgdir(NDTensors)` I've changed this. Thanks!